scripts/build.actions.js

```diff
@@ -33,95 +33,97 @@ class BuildActions extends BaseScript {
    * @memberof DeployActions
    */
   async run (args = [], buildConfig = {}) {
-    const taskName = 'Build actions'
-    this.emit('start', taskName)
+    if (this.config.app.hasBackend) {
```

Comment:
to be consistent here I would do like https://github.com/adobe/aio-app-scripts/blob/master/scripts/deploy.ui.js#L29